### PR TITLE
Correct ages in our privacy policy

### PR DESCRIPTION
**Pull request by islemaster**
Updates our privacy policy to correctly reflect our system's behavior when a student signs one of our petitions.  Before, our policy was inconsistent: In some places it said we anonymize petition signatures for students under age 16, and in other places it says we anonymize them for students under age 13.

This change updates all language to say we anonymize submissions under age 16, which is what our system actually does, as of a policy change in May 2018 to comply with GDPR.

Here's the relevant code:

https://github.com/code-dot-org/code-dot-org/blob/9c537eb5410e0c47d4ded1f6816dcffea505daf7/pegasus/sites.v3/code.org/views/petition_expand.haml#L151-L156

[Here's a oneoff script](https://github.com/code-dot-org/code-dot-org/blob/9c537eb5410e0c47d4ded1f6816dcffea505daf7/bin/oneoff/wipe_data/opt_out_petition_emails_under_16) we wrote at the time of the policy change to opt-out previously gathered signatures from users aged 13-15 from marketing mail.  A comment at the top of that script gives more context on the policy change in question.

## Open questions:

What level of "version bump" does this represent for our privacy policy? See our "Updates to our Privacy Policy" policy at the end of https://code.org/privacy

- Should we update the "Date of last revision" at the top of the policy?
- Do we need to notify users of this change somehow?

### CODE REVIEW

**Comment by dju90** [comment]
Whenever I change the privacy policy, I usually update the "date of last revision". I'm not sure about notifying but I would defer that decision to Alice.

**Comment by islemaster** [comment]
Confirmed with LT: Since this is a correction of details and not a material change in our actual policy, no notification is needed.

**Comment by dju90** [comment]
Hey Brad, sorry this escaped me the first time, but I think you also have to modify the privacy policy here as well for dashboard: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/home/_privacy.md

**Comment by islemaster** [comment]
Oh! I didn't realize we had multiple copies of this.  Updating...

**Comment by dju90** [comment]
Yeah, unfortunately that is the case. Brendan has been a proponent for a long time of consolidating the two so only one file needs to be updated, but I'm not sure what goes into that.

**Comment by breville** [comment]
The updated text discusses users over 16 and under 16 but not actually equal to 16?  (I think we've discussed this before, and maybe one of these cases are for equal 16, but to me it still reads a bit like a gap.)

**Comment by islemaster** [comment]
That problem, if it exists, already existed in the privacy policy as "over the age of 13" and "under the age of 13."  We use "over the age of" to indicate a greater-than-or-equal-to relationship five times in our existing privacy policy.  I think the background for that in legalese is that "age of 13" is referring to the point in time at which one is 13 years old, not the entire year between ages 13 and 14.  So "over the age of 13" includes 13-and-one-day, 13-and-two-days, etc.

File: dashboard/app/views/home/_privacy.md [hunk 1]
```diff
@@ -160,15 +160,15 @@ Code.org does not directly collect or store payment instruments. Donations to Co
...
 
 ### Exceptions
-If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User over the age of 13 signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
+If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User over the age of 16 signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
```
Comment (dju90) [inline dashboard/app/views/home/_privacy.md]:
```suggestion
If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User aged 16 or over signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
```
Checked in GDPR and it says "the processing of the personal data of a child shall be lawful where the child is at
least 16 years old". This language change's grammar is consistent with the grammar we use elsewhere.

File: pegasus/sites.v3/code.org/public/privacy.md [hunk 2]
```diff
@@ -166,15 +166,15 @@ Code.org does not directly collect or store payment instruments. Donations to Co
...
 
 ### Exceptions
-If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User over the age of 13 signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
+If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User over the age of 16 signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
```
Comment (dju90) [inline pegasus/sites.v3/code.org/public/privacy.md]:
```suggestion
If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User aged 16 or over signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
```

**Review by molly-moen — APPROVED** [review]

**Pull request merged**